cloudigrade/account/views.py

```diff
@@ -150,3 +150,30 @@ def list(self, *args, **kwargs):
             'aws_account_id': _get_primary_account_id()
         }
         return Response(response)
+
+
+class CloudAccountOverviewViewSet(viewsets.ReadOnlyModelViewSet):
+    """List all or view a single Cloud Account Overview."""
+
+    queryset = Account.objects.all()
+    serializer_class = serializers.CloudAccountOverviewSerializer
+
+    def list(self, request, *args, **kwargs):
+        """Get overview of accounts filtered to appropriate user."""
+        user = request.user
+        accounts = self.queryset
+        serializer = self.serializer_class(data=request.query_params)
+        serializer.is_valid(raise_exception=True)
+        if not user.is_superuser:
+            accounts = accounts.filter(user=user)
+        user_id = request.query_params.get('user_id', None)
+        if user_id is not None:
+            user_id = convert_param_to_int('user_id', user_id)
+            accounts = accounts.filter(user__id=user_id)
+        overviews = {'cloud_account_overviews': []}
+        # iterate through each account in our queryset and append the
+        # overview to our cloud_account_overviews list
+        for account in accounts:
+            account_overview = serializer.get_overview(account)
+            overviews['cloud_account_overviews'].append(account_overview)
+        return Response(overviews, status=status.HTTP_200_OK)
```

Comment:
Is `status=status.HTTP_200_OK` necessary here? I think the default `status` for a `Response` is already `200` since it inherits (through several layers) from `django.http.response.HttpResponseBase`, but I haven't tested to confirm this.